### ISSUE TITLE
KL loss function on Testing stage

### ISSUE THREAD

**Issue by JIaqiYang78** [slei109/PATNet#27]
Hi, I have a more question about the KL loss function on testing stage. I found that in your paper, there is a unfrozen stage between the support and query test data used KL loss, but in your code->test.py, the 'loss=None'. So, where did you import and use KL loss function? Thanks for you help in advance.

**Comment by slei109** [comment]
The test.py here is to test the performance without fine-tuning step. If you want to fine-tuning the reference layer, you can refer to finetune_reference() function in Line 139 in patnet.py
